src/Cmfd.cpp

```diff
@@ -1311,39 +1166,24 @@ int Cmfd::getCellNext(int cell_id, int surface_id) {
 
 
 /**
- * @brief Return whether optically thick diffusion correction factor is in use.
- * @return Whether optically thick diffusion correction factor is in use.
+ * @brief Return whether Larsen's effective diffusion coefficient correction
+ *        factor is in use.
+ * @return Whether Larsen's effective diffusion coefficient correction factor
+ *         is in use.
  */
-bool Cmfd::isOpticallyThick() {
-  return _optically_thick;
+bool Cmfd::isLarsensEDCOn() {
+  return _larsens_edc_on;
 }
 
 
 /**
- * @brief Set whether optically thick diffusion correction factor is in use.
- * @param optically_thick Boolean indicating whether optically thick diffusion
- *        correction factor is in use.
+ * @brief Set whether Larsen's effective diffusion coefficient correction factor
+ *        is in use.
```

Comment:
It is recommended that Larsen's EDC always be on as it improves stability and, in theory, provides a better approximation of the diffusion coefficient for CMFD-accelerated MOC. This function is nice for testing, but there isn't really any value in the user having this function so I'm just going to take it out completely.